### DISCUSSION TITLE
Quart Fire on my own sql table data Change

### DISCUSSION THREAD

**Discussion by vahidarr** [quartznet/quartznet#674] [General] [answered]
I Have Table in my own Sql database that I want When a Separate Service change a Table Data for example Add new Row to that Table، a trigger or job or anything Fire (base on sql table change) .I dont want check that table in a repetitive time because it is not have a good performance.we have SqlDependency in ADO.NET but i prefer use Quartz.
How can i Do That in Quartz???

**Comment by lahma** [comment]
How can relational database have bad performance for queries given that you have proper indexes in place? The quartz way is to poll for unprocessed changes, process them and and some marker about things already processed.

**Comment by vahidarr** [comment]
the scenario is this:
 immediately after a person added in another service sth must fire and do this job:
 create user account, send sms and email.
I don't think this happen so often but the job after that must done immediately.
is there any way to create a job for 30 second later than now or server to client call or any other ways???

**Comment by lahma** [comment, marked as answer]
You can always schedule a job from outside (scheduler that is not started, ZeroSizeThreadPool, pointing to Quartz database) with trigger data map that contains target id, like user id or something that allows job to point to single task to run.

So creating a trigger with parameters for fixed job should work I believe.

**Comment by vahidarr** [comment]
plz provide sample or link of sample

**Comment by lahma** [comment]
Closing as my time is limited and I cannot build samples for everything, sorry about that.
